src/test/java/uk/gov/hmcts/divorce/divorcecase/validation/CaseValidationTest.java

```diff
@@ -130,6 +132,25 @@ public void shouldReturnErrorWhenDateIsLessThanOneYearAgo() {
         assertThat(response).isEqualTo(List.of("field" + LESS_THAN_ONE_YEAR_AGO));
     }
 
+    @Test
```

Comment:
might be worth adding the success for 'over 1yr' or 1year1day now.

Reply:
Yeah good shout - I'll add them now.

Reply:
Added